CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add support for multi-tenancy. Refer to [docs](https://getstream.io/chat/docs/multi_tenant_chat/?language=swift) for more info [#295](https://github.com/GetStream/stream-chat-swift/issues/295)
 
 ### 🔄 Changed
+- We don't use unknown user anymore. By default the current user is anonymous (you can check this with `isAnonymous`). Anyway you can't connect without `set(user:token:)` or `setGuestUser(user:)` or `setAnonymousUser()` [#284](https://github.com/GetStream/stream-chat-swift/issues/281).
+- You can subscribe to events as soon as the client is configured. This means that your subscriptions will work until the client disconnect (user login/logout) and until you cancel subscriptions [#284](https://github.com/GetStream/stream-chat-swift/issues/281).
```

Comment:
Why don't we start subscriptions after `set(user:)`? Doesn't make sense to start subscribing before connecting, will open ways for more errors.

Reply:
With changes, it will be doesn't matter. For example if you have a separate part of general UI with channels and messages, for example, app badge icon with an unread count. You no need to worry that you should resubscribe after login.
I saw cases where developers subscribe for `connectionState`, but now they no need to worry to do that only after set user.